dist/XSLoader/Makefile.PL

```diff
@@ -8,6 +8,8 @@
 use strict;
 use warnings;
 
+require 5.006;
```

Comment:
Why not `use v5.6;`?

Reply:
There's not much point in using the vstring form of a version for a 5.6 requirement since 5.6 is the first perl version that recognizes it.

Reply:
Point. But there's no reason to use `require` instead of `use`.

Reply:
IMO, A better bike shed is:

> what should dual life modules be doing to specify minimum version?

I would argue we're REALLY inconsistent with the use of these lines. In all honesty, I'm not convinced they have much value. It'd be good to have a documented standard for how we want to use this in our code. After which we can just make one big change to fix them all.

Reply:
This is not a bike shed, there is a functional difference between the suggestions.

Reply:
`require` is most likely a remnant of history, since 5.003 supported a perl version check using `require` but not `use`.

`use v5.6;` or `use 5.006;` both seem reasonable to me. `v5.6` is more human readable. The only real downside I see is that version conditionals are best done in the form `"$]" >= 5.008001`, so they wouldn't be consistent. That's very minor though.

Reply:
done.